inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -1335,21 +1348,21 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
 
     // always gather to Vmm, compute with Vmm, store with Xmm if scalar
     inline void gather_i32_indices(Vmm vmm_src, const Xbyak::Reg64 &base, int offset, Vmm vmm_indices, int scale,
-                                memory::data_type src_dt, bool is_scalar) {
+                                memory::data_type src_prc, bool is_scalar) {
```

Comment:
As I can see the idea is to get rid of oneDNN's memory::data_type* usage?
If so I propose to apply it for this gather_i32_indices as well.
I mean to change the argument type to Precision and update the calls.